sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -423,7 +423,7 @@ export class MessageSender extends LinkEntity {
   /**
    * Initializes the sender session on the connection.
    */
-  private async _init(options?: SenderOptions): Promise<void> {
+  public async open(options?: SenderOptions): Promise<void> {
```

Comment:
`SenderOptions` here is from `rhea-promise` and is not exported by the Service Bus library. I wonder why you didnt get a warning on the generated report from the api-extractor.

This shouldnt be exposed to the user. I would suggest we have the same private _init() as before and add a new `open()` that calls `_init()` with no arguments

Reply:
Nevermind, just realized that this is not the `open()` that is exposed to the user. That one is from the sender.ts file

Reply:
In that case, what is the reason for the rename here? We use `init()` across the internal classes for senders and receivers, across Event Hubs and Service Bus. I'd prefer if we keep the consistency.

Reply:
Parity and it's also a "public" function now. 

I could rename the others (and might when we get into the same change for the receiver side) but I figured we wouldn't want to do that arbitrarily (yet).

Reply:
Leaving in place, but I'll fix the others to be inline with this naming convention as I do the other changes for the receivers.